spec/spec_helper.rb

```diff
@@ -1,4 +1,7 @@
 ENV["RAILS_ENV"] ||= 'test'
+require 'celluloid/test'
+Celluloid.logger.level = Logger::WARN
+require 'cellect/testing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.